app/src/lib/editors/linux.ts

```diff
@@ -100,7 +109,7 @@ const editors: ILinuxExternalEditor[] = [
     name: 'JetBrains PhpStorm',
     paths: [
       '/snap/bin/phpstorm',
-      '.local/share/JetBrains/Toolbox/scripts/phpstorm',
+      '.local/share/JetBrains/Toolbox/scripts/PhpStorm',
```

Comment:
Is it normal to not use lowercase on that one when all other JetBrains IDE are lowercase?

Reply:
> Is it normal to not use lowercase on that one when all other JetBrains IDE are lowercase?

Yes, the toolbox specifically assigns phpstorm's shell script in uppercases. It has been like this for a while, yet it doesn't seem to ever be fixed : /